vuepress/docs/next/docs/curate/bundle-component-details.md

```diff
@@ -0,0 +1,433 @@
+
+**contents-descriptor.yaml**
+
+    id: NWS650
+    typeCode: NWS
+    description: Dealing with a financial emergency
+    mainGroup: free
+    groups:
+      - free
+    status: PUBLIC
+    attributes:
+      - code: date
+        value: '2020-04-23 00:00:00' # the date should be in ISO-8601 format
+      - code: title
+        values:
+          en: Dealing with a financial emergency
+      - code: subtitle
+        values:
+          en: |
+            <p>How to tackle financial stress</p>
+      - code: body
+        values:
+          en: |
+            <p>For many people, financial concerns are their number-one stress point. Here are 6 ways to help reduce your money stress and get motivated to take control of your finances.</p>
+      - code: img
+        values:
+          en:
+            correlationCode: '651'
+            name: bank_750xx684385064_d0.jpg
+      - code: links
+        elements:
+          - code: links
+            value:
+              symbolicDestination: '#!U;http://www.yoursite.com/!#'
+              destType: 1
+              urlDest: 'http://www.yoursite.com/'
+              pageDest: null
+              contentDest: null
+              resourceDest: null
+            values:
+              en: vostro sito
+          - code: links
+            value:
+              symbolicDestination: '#!U;http://www.yoursite.com/!#'
+              destType: 1
+              urlDest: 'http://www.yoursite.com/'
+            values:
+              en: vostro sito 2
+      - code: attaches
+        elements:
+          - code: attaches
+            values:
+              en:
+                correlationCode: '205'
+                name: Entando_Admin_Console_Overview_4.3.3_EN.pdf
+
+## Categories
+This descriptor contains a list of categories. 
+** categories-descriptor.yaml**
+
+    - code: my-category # Category name
+      parentCode: home # The parent Category; home is the base category
+      titles:
+        it: "Una nuova categoria" # Category name in Italian
+        en: "New category" # Category name in English
+
+## Groups
+This descriptor contains a list of groups.
+**groups-descriptor.yaml**
+
+    - code: my_group # Identifies the group
+      name: "My group" # The name of the group
+
+## Labels
+This descriptor contains a list of labels.
+**lables-descriptor.yaml**
+
+    - key: my-first-label # Identifies the label
+      titles: # The titles on the label
+        it: Vostro Titolo # The title in Italian
+        en: Your Title # The title in English
+
+## Languages
+This descriptor contains a list of languages to enable during the installation process.
+
+**languages-descriptor.yaml**
+
+    - code: en
+      description: English
+    - code: it
+      description: Italian
+
+
+<!--
+## Static Resources
```

Comment:
I think we should activate once we get an answer on the direction, hopefully tomorrow. Or make sure we have a note on it and plan to do it once the code is available to test. That could be safer I suppose.